samples/cpp/benchmark_app/inputs_filling.cpp

```diff
@@ -962,10 +962,14 @@ std::map<std::string, ov::TensorVector> get_tensors_static_case(const std::vecto
     return blobs;
 }
 
-void copy_tensor_data(ov::Tensor& dst, const ov::Tensor& src) {
-    if (src.get_shape() != dst.get_shape() || src.get_byte_size() != dst.get_byte_size()) {
-        throw std::runtime_error(
-            "Source and destination tensors shapes and byte sizes are expected to be equal for data copying.");
+void copy_tensor_data(ov::Tensor& dst, const ov::Tensor& src, bool vpu) {
+    if (src.get_shape() != dst.get_shape()) {
+        if (vpu) {
```

Comment:
Why do you set shapes for vpu?

Reply:
Because of the new issue with the dims that we've discussed here: https://github.com/openvinotoolkit/openvino/pull/16322/files.

We need more time to investigate how to fix it properly on vpux plugin side. And we want to unblock https://github.com/openvinotoolkit/openvino/pull/15693 that also has corresponding PRs in other repositories (VPUX plugin too).

Reply:
I think you need to understand the root cause of the issue instead of introducing new and new WAs.

Reply:
I agree, we want to introduce a proper fix :)
But while we're investigating the problem, we're blocking other people.